### PR TITLE
Performance baselines for ARM snapshot resume latency

### PR DESCRIPTION
**Pull request by dianpopa**
# Reason for This PR

Gather performance baselines for snapshot resume long running perf test on ARM.

## Description of Changes

`[Author TODO: add description of changes.]`

- [ ] This functionality can be added in [`rust-vmm`][1].

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache 2.0 license.

## PR Checklist

`[Author TODO: Meet these criteria.]`
`[Reviewer TODO: Verify that these criteria are met. Request changes if not]`

- [ ] All commits in this PR are signed (`git commit -s`).
- [ ] The issue which led to this PR has a clear conclusion.
- [ ] This PR follows the solution outlined in the related issue.
- [ ] The description of changes is clear and encompassing.
- [ ] Any required documentation changes (code and docs) are included in this PR.
- [ ] Any newly added `unsafe` code is properly documented.
- [ ] Any API changes follow the [Runbook for Firecracker API changes][2].
- [ ] Any user-facing changes are mentioned in `CHANGELOG.md`.
- [ ] All added/changed functionality is tested.

[1]: https://github.com/rust-vmm
[2]: ../docs/api-change-runbook.md

### CODE REVIEW

File: tests/integration_tests/performance/test_snapshot_restore_performance.py [hunk 1]
```diff
@@ -94,11 +94,12 @@ def construct_scratch_drives():
     return list(zip(scratchdisks, disk_files))
 
 
-def default_lambda_consumer(env_id):
+def default_lambda_consumer(env_id, workload):
     """Create a default lambda consumer for the snapshot restore test."""
+    print(env_id)
```
Comment (luminitavoicu) [inline tests/integration_tests/performance/test_snapshot_restore_performance.py:99]: Leftover?

**Review by luminitavoicu — APPROVED** [review]
Looks good!

File: tests/integration_tests/performance/test_snapshot_restore_performance.py (hunk 1 above)
Comment (dianpopa) [reply tests/integration_tests/performance/test_snapshot_restore_performance.py]: Indeed, got rid of it.

**Review by mattschlebusch — APPROVED** [review]
LGTM.

File: tools/parse_baselines/main.py [hunk 2]
```diff
@@ -30,15 +30,15 @@
...
+    "snap_restore_performance": ["test_snap_restore_performance"],
     "network_latency": ["test_network_latency"],
 }
 
 DATA_PARSERS = {
     "vsock_throughput": Iperf3DataParser,
     "network_tcp_throughput": Iperf3DataParser,
     "block_performance": BlockDataParser,
-    "snapshot_restore_performance": SnapshotRestoreDataParser,
+    "snap_restore_performance": SnapshotRestoreDataParser,
```
[the remark this answers is not in the document]
Comment (dianpopa) [reply tools/parse_baselines/main.py:41]: Yeah, that is a good observation. I think we need to do it for all of the tests (they quite repeat throughout themselves). But I would do it out of the scope of this PR.

**Review by luminitavoicu — APPROVED** [review]

**Pull request merged**